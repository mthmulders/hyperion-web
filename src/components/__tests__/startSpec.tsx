import * as React from 'react';

import { shallow } from 'enzyme';

import Start from '../start';

describe('<Start />', () => {
    it('should render', () => {
        // Act
        const wrapper = shallow(<Start />);

        // Assert
        expect(wrapper).toMatchSnapshot();
    });
});
