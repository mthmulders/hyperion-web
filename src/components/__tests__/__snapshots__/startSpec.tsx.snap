// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Start /> should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Start />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Welcome to Υπερίων
        </h1>,
        <p>
          Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch "Smart Meter" Requirements, or DSMR for short.
        </p>,
        <p>
          Use the above menu to get started.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to Υπερίων",
        },
        "ref": null,
        "rendered": "Welcome to Υπερίων",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch \\"Smart Meter\\" Requirements, or DSMR for short.",
        },
        "ref": null,
        "rendered": "Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch \\"Smart Meter\\" Requirements, or DSMR for short.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Use the above menu to get started.",
        },
        "ref": null,
        "rendered": "Use the above menu to get started.",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Welcome to Υπερίων
          </h1>,
          <p>
            Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch "Smart Meter" Requirements, or DSMR for short.
          </p>,
          <p>
            Use the above menu to get started.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to Υπερίων",
          },
          "ref": null,
          "rendered": "Welcome to Υπερίων",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch \\"Smart Meter\\" Requirements, or DSMR for short.",
          },
          "ref": null,
          "rendered": "Υπερίων is a web application to monitor a 'Smart Energy Meter' that follows the Dutch \\"Smart Meter\\" Requirements, or DSMR for short.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Use the above menu to get started.",
          },
          "ref": null,
          "rendered": "Use the above menu to get started.",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
